Aptoide/aptoide-client-v8
# [ASV-2222] Login View

**What does this PR do?**

   This PR aims implementing the dark theme view for the login pages. Also took the chance to fix the landscape layouts (they had a different view from the portrait ones).

**Database changed?**

No

**Where should the reviewer start?**

- [ ] fragment_login_signup.xml

**How should this be manually tested?**

 Open the app in a device with darktheme and see the login views. Also check it with the light theme to make sure nothing was broken. Don't forget testing in landscape.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2222](https://aptoide.atlassian.net/browse/ASV-2222)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
